src/ansiblelint/rules/name.md

```diff
@@ -21,12 +21,21 @@ If you want to ignore some of the messages above, you can add any of them to the
 
 ## name[prefix]
 
-This rule applies only to included task files that are not named `main.yml`. It
-suggests adding the stem of the file as a prefix to the task name.
+This rule applies only to included task files that are not named `main.yml` or
```

Comment:
I do think this sentence can be a tad confusing until the next two paragraphs are read. I am open to suggestions here.

Reply:
@cidrblock WDYT?

I personally do not have a strong opinion here, but I kinda like the idea of not forcing people to add `| main |` extra prefix for `tasks/mains.yml` which is like 95% of them.